plugins/aea-ledger-cosmos/setup.py

```diff
@@ -41,9 +41,9 @@
     install_requires=[
         "open-aea>=1.0.0, <2.0.0",
         "ecdsa>=0.15,<0.17.0",
-        "bech32==1.2.0",
+        "bech32>=1.2.0,<2",
         "pycryptodome>=3.10.1,<4.0.0",
-        "cosmpy==0.9.2",
+        "cosmpy>0.9.2,<1",
```

Comment:
https://github.com/valory-xyz/open-aea/actions/runs/8734578406/job/24148746278?pr=736#step:6:36

Also this plugin is frequently updated with the latest releases of the cosmpy so no need to set a loose dependency range for cosmpy, because it might lead to the plugin breaking if the cosmpy contains a breaking change

Reply:
True, it's not stable version yet so I guess it shouldn't be dynamic. Fixed back!